pages.fr/common/git-commit.md

```diff
@@ -0,0 +1,21 @@
+# git commit
+
+> Commit les fichers dans le repository.
+> Plus d'informations : <https://git-scm.com/docs/git-commit>.
+
+- Commit les fichiers en stage dans le repository avec un message :
```

Comment:
Use "fichiers indexés" rather than "fichiers en stage", as you did on the "git add" page

Reply:
What should we use in french for `commit` ? `enregistre` maybe ?

Reply:
Personaly, I think `commit` is appropriate. This is the term used in the French documentation of git (https://git-scm.com/docs/git-commit/fr)